pkg/crc/config/settings.go

```diff
@@ -70,7 +70,7 @@ func RegisterSettings(cfg *Config) {
 
 	// Preset setting should be on top because CPUs/Memory config depend on it.
 	cfg.AddSetting(Preset, string(preset.OpenShift), validatePreset, RequiresDeleteAndSetupMsg,
-		fmt.Sprintf("Virtual machine preset (valid values are: %s, %s and %s)", preset.Podman, preset.OpenShift, preset.OKD))
+		fmt.Sprintf("Virtual machine preset (valid values are: %s, %s, %s and %s)", preset.Podman, preset.OpenShift, preset.Microshift, preset.OKD))
```

Comment:
A more future-proof alternative is to add `func preset.AllPresets() []preset.Preset` and use this in `RegisterSettings` and `validatePreset`